global.json

```diff
@@ -10,7 +10,8 @@
     "src/ResourceManagement/Graph.RBAC/Microsoft.Azure.Graph.RBAC",
     "src/ResourceManagement/DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics",
     "src/ResourceManagement/DataLake.Store/Microsoft.Azure.Management.DataLake.Store",
-    "src/ResourceManagement/DataLake.StoreUploader/Microsoft.Azure.Management.DataLake.StoreUploader"
+    "src/ResourceManagement/DataLake.StoreUploader/Microsoft.Azure.Management.DataLake.StoreUploader",
+    "src/ResourceManagement/DataLake.StoreUploader/Microsoft.Azure.Management.RemoteApp"
```

Comment:
shouldn't this be src/ResourceManagement/RemoteApp/Microsoft.Azure.Management.RemoteApp?